code/AdobeEdge.brs

```diff
@@ -1075,7 +1074,9 @@ function _adb_EdgeRequestWorker(stateManager as object) as object
             url = _adb_buildEdgeRequestURL(configId, requestId, edgeDomain)
             _adb_log_verbose("request JSON: " + FormatJson(jsonBody))
             response = _adb_serviceProvider().networkService.syncPostRequest(url, jsonBody)
-            response.requestId = requestId
```

Comment:
Let's add a helper to generate identity map. That helper should have access to statemanager.

Reply:
It's not a good time to refactor it now, we can do it when supporting other `identities` in the xdm request.

Reply:
Agreed